service/src/main/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelper.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.Respondent;
+import uk.gov.hmcts.reform.fpl.model.RespondentParty;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.Objects;
+
+import static org.apache.commons.lang3.ObjectUtils.defaultIfNull;
+import static org.apache.commons.lang3.StringUtils.isNoneBlank;
+import static org.springframework.util.CollectionUtils.isEmpty;
+
+public class EmailNotificationHelper {
+
+    private EmailNotificationHelper() {
+    }
+
+    public static String buildSubjectLine(final CaseData caseData) {
+        final String lastName = getRespondents1Lastname(caseData);
+        final String familyManCaseNumber = defaultIfNull(caseData.getFamilyManCaseNumber(), "");
+
+        return String.format("%1$s%2$s%3$s", lastName,
```

Comment:
Why not simply, Stream.of(lastName, familyManCaseNumber).collect(joining(",")?

Reply:
Or remove 1$, as we are using them in order they are just cluttering.

Reply:
updated